pcdsdevices/sim.py

```diff
@@ -41,6 +41,12 @@ def __init__(self, *args, init_pos=0, **kwargs):
             kwargs.pop(kw, None)
         super().__init__(init_pos=init_pos, **kwargs)
 
+    def set_current_position(self, position):
+        """
+        Adjust offset until the current position is the input position.
```

Comment:
FastMotors don't have offsets, right?

Reply:
Also, was this put here as a start for adding `set_current_position` to `SyncAxesBase` as well?

Reply:
I don't remember why I put this here